polytorch/data.py

```diff
@@ -1,19 +1,27 @@
 from torch import nn
+from typing import List
 from dataclasses import dataclass
 
 from .enums import ContinuousDataLossType
 
 class PolyData():
+    name:str = ""
+
     def create_module(self, embedding_size:int):
         raise NotImplementedError(f'Use either CategoricalData, OrdinalData or ContinuousData')
```

Comment:
No f-string required here or in `size`.

Reply:
just a habit I guess :)

Reply:
I've become used to `ruff` complaining to me about it :)